services/dect-wip/app.py

```diff
@@ -294,6 +294,13 @@ def fetch_default_data_for_templates():
     # init flask
     app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + database_name
     app.secret_key = config['flask'].get('secret_key')
+    if not app.secret_key:
+        app.secret_key = utilities.getRandomStr(16)
```

Comment:
Weeeeell, [`random`](https://docs.python.org/3/library/random.html) as used by the `utilities` module isn't meant for security purposes… Flask recommends [`secrets.token_hex()`](https://flask.palletsprojects.com/en/stable/config/#SECRET_KEY) instead.